src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -662,7 +662,7 @@ protected function getContainerBuilder()
     {
         $container = new ContainerBuilder(new ParameterBag($this->getKernelParameters()));
 
-        if (class_exists('ProxyManager\Configuration')) {
+        if (class_exists('ProxyManager\Configuration') && class_exists('Symfony\Bridge\ProxyManager\LazyProxy\Instantiator\RuntimeInstantiator')) {
```

Comment:
but I'm not sure If that makes any sense, if class ```RuntimeInstantiator``` exist then ```ProxyManager\Configuration``` exist also https://github.com/hason/symfony/blob/pm/src/Symfony/Bridge/ProxyManager/composer.json#L21 without checking it